src/IdePurescript/Completion.purs

```diff
@@ -80,7 +79,7 @@ getSuggestions port
     case match' explicitImportRegex line of
       Just [ Just _, Just mod, Just token ] -> do
         let cc ns = Tuple ns <$> getCompletion' Nothing [C.PrefixFilter token, C.NamespaceFilter [ ns ] ] port mainModule Nothing [ mod ] getQualifiedModule opts
-        completions <- traverse cc [ C.NSValue, C.NSType, C.NSKind ]
+        completions <- traverse cc [ C.NSValue, C.NSType ]
```

Comment:
So this is problematic. It will only work without kind on 0.14 but it will not give great results without it on prior versions. That's why I was thinking it'd be ideal if psc-ide just leniently parsed Kind as Type. Otherwise we might need a way to differentiate versions here.

Reply:
Might be easiest for now to actually take this as an argument to `getSuggestions`.

Reply:
I'm happy to merge and then look at improving for 0.13, I think it should probably already behave differently anyway

Reply:
I think we should just drop the namespace filter here. Filtering by _all_ namespaces is the same as not filtering at all.

Reply:
That's not what this is doing. This is making separate requests for the different namespaces, in order to enrich the completions with namespaces, because prior to 0.14.0 that information was not on the completion response objects. I think this would have to depend on version, with 0.13 make 3 requests for those namespaces, and on 0.14 it should only make 1 request and use the new field. 

But I am happy to look at making that change after merging, because this version should at least work (apart from leaving 0.13 users with no kinds)

Reply:
Doh, thanks for clearing that up. I agree that sounds like the way forward for now.

Reply:
I didn't think about the option of just leaving 0.13 users with no kinds. Seems very reasonable given the fact they can very easily use an older version (at least on VSCode).